### DISCUSSION TITLE
AzuraCast Advertising at end of song

### DISCUSSION THREAD

**Discussion by kiRyyy** [AzuraCast/AzuraCast#4855] [Q&A] [answered]
Hello,
I just installed AzuraCast and I have one music repeating itself for testing purposes, and sometimes I hear this advertise "you are now listening to Azuracast... bla bla" at the end of the song. How to take it off?

**Comment by BusterNeece** [comment, marked as answer]
@kiRyyy Usually adding a second song fixes this issue. AzuraCast isn't really well-suited to repeating one song over and over, as it depends on the transitions between songs to know when one ends and the other begins.
